### ISSUE TITLE
Compliance

### ISSUE THREAD

**Issue by JerryCasiano** [orschiro/simple-translate#7]
I hate to point this out, but unfortunately it's something that will need to be addressed.

The reason I hate to point it out is two fold.

For one, the app is young.

For two, it's pretty onerous, at least in my opinion. Yandex requires in license that they be credited in several areas of the application, including the main window. They also require that the font be the same size and color as the rest of the content.

Now, personally I think it's probably not worth worrying about at this point and believe it's better for both parties to wait till the app matures a little.

But definitely need to start thinking about this since it could cause issues for the project.
Need to find a tasteful way of complying. Maybe even request that they relax the requirement a bit. Not sure how receptive to that such a large company would be but there's no harm in asking, if for example a lighter font color would be acceptable in the main interface.

Anyway, thoughts?

**Comment by Grabli66** [comment]
I understand your concern. I thought about this problem year ago. But i don't have any solution of that. I think there are no good free translating service. Sometimes i thought about my own translating service, but it's too difficult, and need a lot of work and money. If your have any thoughts how to solve this problem, please tell us.

**Comment by orschiro** [comment]
Dear @JerryCasiano,

> Need to find a tasteful way of complying. Maybe even request that they relax the requirement a bit. Not sure how receptive to that such a large company would be but there's no harm in asking, if for example a lighter font color would be acceptable in the main interface.

Thanks for raising this concern! 

I would propose to continue developing and maturing the app and openly approach Yandex in the near future to be transparent about the issues we see and the hope for finding ways to find a solution for mutual benefits. 

Warmly,

~Robert

**Comment by JerryCasiano** [comment]
Agreed, I think it's important for both parties that it be a polished experience.

I think I'll start by adding an about dialog crediting them. It's a start.

**Comment by orschiro** [comment]
> I think I'll start by adding an about dialog crediting them. It's a start.

Great! Do you want some input from @Grabli66 or me?

~Robert

**Comment by JerryCasiano** [comment]
Sure.

It'll be just the standard GTK+ about dialog. However, it would be good to know anyone who should be credited such as authors, translators, artists, etc.

**Comment by orschiro** [comment]
For now I can just think of you and @Grabli66 for the programming. 

Anyone else @Grabli66 you can think of?
